interface/modules/custom_modules/oe-module-dorn/public/primary_config.php

```diff
@@ -0,0 +1,183 @@
+ * @link    http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+    require_once "../../../../globals.php";
+
+    use OpenEMR\Common\Acl\AclMain;
+    use OpenEMR\Common\Csrf\CsrfUtils;
+    use OpenEMR\Common\Twig\TwigContainer;
+    use OpenEMR\Modules\Dorn\ConnectorApi;
+    use OpenEMR\Core\Header; //this is needed along with setupHeader() to get the pop up to appear
+
+    $tab = "Configure Primary";
+
+if (!AclMain::aclCheckCore('admin', 'users')) {
+    echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => xl("Edit/Add Procedure Provider")]);
+    exit;
+}
+
+if (!empty($_POST)) {
+    if (isset($_POST['SubmitButton'])) { //check if form was submitted
+        $datas = ConnectorApi::getPrimaryInfos($_POST['npi']);
+        if ($datas == null) {
+            $datas = [];
+        }
+    }
+}
+
+
+?>
+<html>
+    <head>
+        <?php Header::setupHeader(); ?>
+        <link rel="stylesheet" href="../../../../../public/assets/bootstrap/dist/css/bootstrap.min.css">
+    </head>
+<title> <?php echo xlt("DORN Configuration"); ?>  </title>
+<script>
+    // Process click to pop up the add window.
+function doedclick_edit(npi) {
+    top.restoreSession();
+    var addTitle = '<i class="fa fa-plus" style="width:20px;" aria-hidden="true"></i> ' + <?php echo xlj("Edit Mode"); ?>;
+    let scriptTitle = 'primary_config_edit.php?npi=' + npi +'&csrf_token_form=' + <?php echo js_url(CsrfUtils::collectCsrfToken()); ?>;
+    dlgopen(scriptTitle, '_blank', 800, 750, false, addTitle);
+}
+function doedclick_add() {
+    top.restoreSession();
+    var addTitle = '<i class="fa fa-plus" style="width:20px;" aria-hidden="true"></i> ' + <?php echo xlj("Edit Mode"); ?>;
+    let scriptTitle = 'primary_config_edit.php?csrf_token_form=' + <?php echo js_url(CsrfUtils::collectCsrfToken()); ?>;
+    dlgopen(scriptTitle, '_blank', 800, 750, false, addTitle);
+}
+</script>
+<body>
+    <div class="row"> 
+        <div class="col">
+            <?php
+                require '../templates/navbar.php';
+            ?>
+        </div>
+    </div>
+    <div class="row"> 
+        <div class="col">
+            <h1><?php echo xlt("DORN - Primary Setup"); ?></h1>        
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <div class="card">
+                <div class="row">
+                    <div class="col">
+                        <p>
+                            The DORN network requires basic address setup for your clinic. This setup is based on NPI. The first and default entry will be your billing NPI. If there are other NPI's in your clinic that have their own account at a lab, 
+                            then more than 1 entry can be created here.
```

Comment:
rec translating above with xlt(). include the entire thing in one xlt() call and keep it all on one line.